var/spack/repos/builtin/packages/eztrace/package.py

```diff
@@ -15,7 +15,7 @@ class Eztrace(Package):
     git = "https://gitlab.com/eztrace/eztrace.git"
 
     version("master", branch="master")
-    version("2.0-rc1", sha256="2ad322df5aeb0c668ebadbfeb3b21c8841831b2c15944d86d1d217f738fee74e")
+    version("2.0", sha256="899788d67e651bbe54abf39099f8f28a843d6676")
```

Comment:
How did you calculate that?  I get the following from `spack checksum`:

```suggestion
    version("2.0", sha256="67bd296f059cdfab303c62f674af3e1e858213d6945bd79cb8ede4a035c0c2d6")
```